test/donotoptimize_assembly_test.cc

```diff
@@ -38,88 +38,100 @@ const ExtraLarge ConstExtraLargeObj{};
 // CHECK-LABEL: test_with_rvalue:
 extern "C" void test_with_rvalue() {
   benchmark::DoNotOptimize(Add42(0));
-  // CHECK: movl $42, %eax
-  // CHECK: ret
+  // CHECK-CLANG: mov 2a,REG
+  // CHECK-CLANG: ret
 }
 
 // CHECK-LABEL: test_with_large_rvalue:
 extern "C" void test_with_large_rvalue() {
   benchmark::DoNotOptimize(Large{ExternInt, {ExternInt, ExternInt}});
-  // CHECK: ExternInt(%rip)
```

Comment:
Note that we lose the `ExternInt` label but that does not affect the goal of the test.